### PR TITLE
MRG: Fix CircleCI

### PR DESCRIPTION
**Pull request by larsoner**
Todo:

- [x] fix plot_covariance_whitening_dspm.py
- [x] fix plot_lcmv_beamformer_volume.py
- [x] get rid of mysterious `*2` factor we had to add to our tolerance

The second one is not straightforward, so I'll probably tackle it Monday.

### CODE REVIEW

File: examples/inverse/plot_covariance_whitening_dspm.py [hunk 1]
```diff
@@ -118,7 +118,7 @@
     for cov in noise_covs:
         inverse_operator = make_inverse_operator(evokeds[0].info, forward,
                                                  cov, loose=0.2, depth=0.8,
-                                                 rank=274)
+                                                 rank=dict(meg=274))
```
Comment (agramfort) [inline examples/inverse/plot_covariance_whitening_dspm.py]: any idea why we need to specify it manually? it should not be necessary here I think

**Comment by larsoner** [comment]
Okay I figured out the problem was that we were not applying projectors when calculating the rank. Now we do by default (with an expert option not to). Doing this allowed me to get rid of the mysterious `*2` factor that we had to have in our `_estimate_rank_from_s` function (hooray!).

**Comment by larsoner** [comment]
https://11369-1301584-gh.circle-artifacts.com/0/dev/auto_examples/inverse/plot_lcmv_beamformer_volume.html

https://11369-1301584-gh.circle-artifacts.com/0/dev/auto_examples/inverse/plot_covariance_whitening_dspm.html

**Comment by agramfort** [comment]
@larsoner please merge if you're done here. LGTM thanks

**Pull request merged**